vars/iqeUtils.groovy

```diff
@@ -502,7 +543,7 @@ def configIQE(String appName, Map options) {
 
     writeVaultEnvVars(options)
     options['extraEnvVars'].each { key, value ->
-        writeEnv(key, value)
+        writeEnv(key, value instanceof Closure ? value(env) : value)
```

Comment:
Can you show an example of when you need to pass a closure into 'extraEnvVars'? I was expecting these to simply be plain strings.

Reply:
https://github.com/RedHatInsights/iqe-jenkins/blob/master/pipelines/imageBuilderProdTestSuite.groovy#L21

IIRC we couldn't use `${env.WORKSPACE}` directly b/c at the time when this line is evaluated the value was undefined.